src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -138,6 +138,14 @@ public function getServerParameter(string $key, mixed $default = ''): mixed
         return $this->server[$key] ?? $default;
     }
 
+    /**
+     * Gets server parameters.
```

Comment:
Comment not needed

Reply:
All methods in the class have comments. Don't we need to be consistent with the CS of the file ?